src/components/common/animations/SaveButton/SaveButton.js

```diff
@@ -42,7 +43,7 @@ class SaveButton extends AnimationBase {
     }
   }
 
-  handlePress = () => {
+  handlePress = useOnPress(() => {
```

Comment:
hooks haven't be used in class components, we've discussed this on the last sync.
i've asked for now just .preventDefault - anyway in future we won't have any class components

```suggestion
  handlePress = e => {
   e.preventDefault()
```

Reply:
done